packages/structure/src/model/RWRoute.ts

```diff
@@ -49,7 +49,7 @@ export class RWRoute extends BaseNode {
       ?.getOpeningElement()
       ?.getTagNameNode()
       ?.getText()
-    return tagText === 'Private'
+    return tagText === 'Private' || tagText === 'PrivateSet'
```

Comment:
Do we really need to keep "Private"? I think it has to be "PrivateSet" now.

Reply:
I would say lets remove `Private` in a separate PR, and remove it from exports too!